### PR TITLE
Remove support for Android 6

### LINKED ISSUE #534: Remove Android 6 support

### PR DESCRIPTION
**Pull request by jmg-duarte**
closes #534

We should probably do a new release with this

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -492,7 +492,6 @@ $(soTargetPatterns6): cargo-init make-always
 	  -u builder \
 	  -w /src/rust/actyx \
 	  -e HOME=/home/builder \
-	  -e ANDROID6=yes \
```
Comment (rkuhn) [inline Makefile:495]: the whole soTargetPatterns6 needs to go, removing .so files from the APKs as a result (making them smaller)

**Review by rkuhn — COMMENTED** [review]
I don’t see a change to `node-ffi/build.rs`, perhaps github can’t handle that many file deletions?

**Comment by jmg-duarte** [comment]
> I don’t see a change to `node-ffi/build.rs`, perhaps github can’t handle that many file deletions?

I removed it completely, from my understanding, it was completely related to Android 6. But here's "the diff":
```diff
- fn main() {
-     println!("cargo:rerun-if-env-changed=ANDROID6");
-     if std::env::var("ANDROID6").is_ok() {
-         cc::Build::new()
-             .cargo_metadata(false)
-             .include("android_headers")
-             .file("ifaddrs.c")
-             .compile("ifaddrs");
-         println!("cargo:rustc-link-lib=static:+whole-archive=ifaddrs");
-         println!(
-             "cargo:rustc-link-search=native={}",
-             std::env::var("OUT_DIR").expect("cannot build in non-UTF8 target directory")
-         );
-     }
- }
```

File: Makefile (hunk 1 above)
Comment (jmg-duarte) [reply Makefile:495]: Handled in https://github.com/Actyx/Actyx/pull/538/commits/897edc2fdbc87e894ccff6132486ca19c7b9ea4d

**Review by rkuhn — APPROVED** [review]

**Pull request merged**